lib/taskana-core/src/test/java/acceptance/task/CompleteTaskAccTest.java

```diff
@@ -206,62 +242,73 @@ void testForceClaimTaskFromOtherUser() throws Exception {
 
   @WithAccessId(user = "user-1-1")
   @Test
-  void testClaimTaskNotExisting() {
+  void should_ThrowException_When_ClaimingNonExistingTask() {
     assertThatThrownBy(() -> taskService.claim("NOT_EXISTING"))
         .isInstanceOf(TaskNotFoundException.class);
   }
 
-  @WithAccessId(user = "user-1-2")
+  @WithAccessId(user = "user-1-1")
   @Test
-  void testClaimTaskWithInvalidState() {
-    assertThatThrownBy(() -> taskService.forceClaim("TKI:000000000000000000000000000000000036"))
+  void should_ThrowException_When_ClaimingTaskInInvalidState() throws Exception {
+    Task task =
+        createDefaultTask()
+            .state(TaskState.COMPLETED)
+            .completed(Instant.now())
+            .buildAndStore(taskService);
+    assertThatThrownBy(() -> taskService.forceClaim(task.getId()))
         .isInstanceOf(InvalidStateException.class);
   }
 
-  @WithAccessId(user = "user-1-2")
+  @WithAccessId(user = "user-1-1")
   @Test
-  void testClaimTaskWithInvalidOwner() {
-    assertThatThrownBy(() -> taskService.claim("TKI:000000000000000000000000000000000035"))
+  void should_ThrowException_When_ClaimingTaskWithInvalidOwner() throws Exception {
+    Task task =
+        createDefaultTask()
+            .owner("user-1-2")
+            .state(TaskState.CLAIMED)
+            .claimed(Instant.now())
+            .buildAndStore(taskService);
+    assertThatThrownBy(() -> taskService.claim(task.getId()))
         .isInstanceOf(InvalidOwnerException.class);
   }
 
-  @WithAccessId(user = "user-1-2")
+  @WithAccessId(user = "user-1-1")
   @Test
-  void testCancelClaimForcedWithInvalidState() {
-    ThrowingCallable call =
-        () -> taskService.forceCancelClaim("TKI:000000000000000000000000000000000036");
+  void should_ThrowException_When_ForceCancelClaimTaskWithInvalidState() throws Exception {
+    Task task =
+        createDefaultTask()
+            .state(TaskState.COMPLETED)
+            .completed(Instant.now())
+            .buildAndStore(taskService);
+    ThrowingCallable call = () -> taskService.forceCancelClaim(task.getId());
```

Comment:
> Please extract the lambda statement, so that we use the given, when, then theorem :)
And don't forget to add empty lines between each block. Thanks!